ts/features/idpay/initiative/details/screens/OperationsListScreen.tsx

```diff
@@ -0,0 +1,172 @@
+//       circuitType: "MASTERCARD",
+//       maskedPan: "1234",
+//       operationId: "7"
+//     },
+//     {
+//       operationType: OnboardingOperationTypeEnum.ONBOARDING,
+//       operationDate: new Date(),
+//       operationId: "0"
+//     },
+//     {
+//       operationType: IbanOperationTypeEnum.ADD_IBAN,
+//       operationDate: new Date(),
+//       iban: "IT1234567890123456789012345",
+//       operationId: "1",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.ADD_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       maskedPan: "1234",
+//       operationId: "2",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.ADD_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_visa.png",
+//       maskedPan: "1234",
+//       operationId: "3",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.DELETE_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       maskedPan: "1234",
+//       operationId: "4",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.DELETE_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_visa.png",
+//       maskedPan: "1234",
+//       operationId: "5",
+//       channel: "APP"
+//     }
+//   ]
+// };
+
+export const OperationsListScreen = () => {
+  const route = useRoute<RouteProps>();
+  const { initiativeId } = route.params;
+  const dispatch = useIODispatch();
+
+  const timelineFromSelector = useIOSelector(idpayTimelineSelector);
+  const timelinePageFromSelector = useIOSelector(idpayTimelinePageSelector);
+  const isTimelineLoading = pot.isLoading(timelineFromSelector);
+  const timelineSome = pot.getOrElse(timelineFromSelector, {
+    lastUpdate: new Date(),
+    operationList: []
+  });
+  const isTimelinePageLoading = pot.isLoading(timelinePageFromSelector);
+  const fetchMoreItems = () => {
+    if (!pot.isError(timelinePageFromSelector) && !isTimelinePageLoading) {
+      dispatch(idpayTimelinePaginationGet.request({ initiativeId }));
+    }
+  };
+
+  const renderContent = () => (
+    <View style={IOStyles.horizontalContentPadding}>
+      <H1>Lista operazioni</H1>
+      <Body>
+        Ultimo aggiornamento:{" "}
+        <Body weight="SemiBold">
+          {formatDateAsLocal(timelineSome.lastUpdate, true)}
+        </Body>
+      </Body>
+      <NBView spacer large />
+      <FlatList
+        style={{ height: "100%" }}
```

Comment:
```suggestion
        style={[{ height: "100%" }, IOStyles.horizontalContentPadding]}
```
There is a padding inside the flat list scrollbar, this (plus the comment above) should resolve the problem I think.